discovery-infra/start_discovery.py

```diff
@@ -69,8 +69,9 @@ def create_nodes_and_wait_till_registered(inventory_client, cluster, image_path,
                  sleep_seconds=5, waiting_for="Nodes to be registered in inventory service")
     log.info("Registered nodes are:")
     pprint.pprint(inventory_client.get_cluster_hosts(cluster.id))
-    utils.wait_till_all_hosts_are_in_status(client=inventory_client, cluster_id=cluster.id,
-                                            nodes_count=nodes_count, status=consts.NodesStatus.KNOWN)
+    if not inventory_client:
```

Comment:
why this if is needed?

Reply:
you have the case of checking if client exist and then doing updates etc', so i am keeping in line. I guess you wanted to support flow when image already created and we just want to boot VMs.

Reply:
it will not even get here

Reply:
yap, but u don't need this line look on line 62